adapter/PreciceInterface.c

```diff
@@ -556,7 +556,7 @@ void PreciceInterface_ConfigureNodesMesh( PreciceInterface * interface, Simulati
         count += 1;
       }
     }
-    assert (count == interface->num2DNodes);
+    assert (count == interface->num2DNodes && "Filtering of 2D nodes not done properly. Please make sure the out-of-plane axis is Z-axis");
```

Comment:
Does this really prevent the misuse of CalculiX in 2D in, e.g. the x-y plane, or is it just a hint?